File: app/src/main/java/com/suribetpos/main/data/model/topup/TopUpPaidCommissionDetails.java
package com.suribetpos.main.data.model.topup;

/**
 * Created by DEV 27 on 16/02/2017.
 */

public class TopUpPaidCommissionDetails {
    public double getSalesCommission() {
        return SalesCommission;
    }

    public void setSalesCommission(double salesCommission) {
        SalesCommission = salesCommission;
    }

    public double getTotalSold() {
        return TotalSold;
    }

    public void setTotalSold(double totalSold) {
        TotalSold = totalSold;
    }

    public double getLocationSoldCommission() {
        return LocationSoldCommission;
    }

    public void setLocationSoldCommission(double locationSoldCommission) {
        LocationSoldCommission = locationSoldCommission;
    }

    public double getLocationBalance() {
        return LocationBalance;
    }

    public void setLocationBalance(double locationBalance) {
        LocationBalance = locationBalance;
    }

    public double getCancelAmount() {
        return CancelAmount;
    }

    public void setCancelAmount(double cancelAmount) {
        CancelAmount = cancelAmount;
    }

    public int getCancelCount() {
        return CancelCount;
    }

    public void setCancelCount(int cancelCount) {
        CancelCount = cancelCount;
    }

    double SalesCommission,TotalSold,LocationSoldCommission, LocationBalance, CancelAmount;
    int CancelCount;
}

File: app/src/main/java/com/suribetpos/main/ui/commission/model/TopUpPaidCommissionDetails.kt
package com.suribetpos.main.ui.commission.model

/**
 * Created by DEV 27 on 16/02/2017.
 */
data class TopUpPaidCommissionDetails (
    var SalesCommission:Double = 0.0,
    var TotalSold :Double = 0.0,
    var LocationSoldCommission :Double = 0.0,
    var LocationBalance :Double = 0.0,
    var CancelAmount :Double = 0.0,
    var CancelCount:Double = 0.0
)